extensions/markdown-language-features/src/commands/openImage.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Command } from '../commandManager';
+import { MarkdownPreviewManager } from '../preview/previewManager';
+
+export class OpenImageCommand implements Command {
+	public readonly id = '_markdown.openImage';
+	public constructor(
+		private readonly _webviewManager: MarkdownPreviewManager,
+		private _currentPanel: vscode.WebviewPanel | undefined = undefined
+	) { }
+
+	public execute(args: { src: string; id: string }) {
+		const columnToShowIn = vscode.window.activeTextEditor
```

Comment:
Can this be more similar to `CopyImageCommand`? I don't think the extra logic calling `reveal`/`create` is needed